composer.json

```diff
@@ -20,7 +15,7 @@
         "zendframework/zend-json": "2.2.*"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.8.*"
+        "phpunit/phpunit": "*"
```

Comment:
not recommended to open it up like that, create a starting version, even if it's just the latest. Why it's not recommended is that your unit tests will not work on lower versions like `4.8` as they are not forward compatible with PSR-4.